e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -107,10 +107,16 @@ public async Task RegistryManager_AddDeviceWithTwinWithDeviceCapabilities()
 
             await registryManager.AddDeviceWithTwinAsync(iotEdgeDevice, twin).ConfigureAwait(false);
 
-            Device actual = await registryManager.GetDeviceAsync(deviceId).ConfigureAwait(false);
-            await registryManager.RemoveDeviceAsync(deviceId).ConfigureAwait(false);
-
-            Assert.IsTrue(actual.Capabilities.IotEdge);
+            try
+            {
+                Device actual = await registryManager.GetDeviceAsync(deviceId).ConfigureAwait(false);
+                actual.Should().NotBeNull($"Got null in GET on device {deviceId} to check IotEdge property.");
```

Comment:
I think it is null because of a lag issue with the device registry. I'm going to aim to solve that independently, but in the meantime, I'd like not to see NREs in test results.